lib/urlwatch/util.py

```diff
@@ -99,3 +102,12 @@ def edit_file(filename):
         raise SystemExit('Please set $VISUAL or $EDITOR.')
 
     subprocess.check_call(shlex.split(editor) + [filename])
+
+
+def import_module_from_source(module_name, source_path):
+    loader = importlib.machinery.SourceFileLoader(module_name, source_path)
+    spec = importlib.util.spec_from_file_location(module_name, source_path, loader=loader)
```

Comment:
Explicitly specifying a loader to allow arbitrary file extension.
E.g. `share/urlwatch/examples/hooks.py.example`
Otherwise `spec` would be `None` on unsupported extensions.